package.json

```diff
@@ -83,6 +83,7 @@
         }
       ],
       "react/jsx-one-expression-per-line": "off",
+      "react/jsx-props-no-spreading": "off",
```

Comment:
Rather than this I recommend disabling for _just_ the `DateField` file and explaining why it's disabled

Reply:
This was the old version. What I ended up doing was  setting ` "explicitSpread": "ignore"`